openflow13/match.go

```diff
@@ -1482,6 +1483,41 @@ func NewTunnelIpv6DstField(tunnelIpv6Dst net.IP, tunnelIpv6DstMask *net.IP) *Mat
 	return f
 }
 
+// TtlField TTL field
```

Comment:
The comment doesn't look helpful and is wrong grammatically.
Maybe either remove it or comment `NXM_NX_IP_TTL field`

Reply:
Done. I removed the inappropriate comment. Can you help review this PR again? Thanks!